.gitignore

```diff
@@ -1,2 +1,4 @@
 node_modules
 dist
+.yarn/cache
```

Comment:
I put them here because of – https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored